app/controllers/disputes/appeals_controller.rb

```diff
@@ -10,6 +10,8 @@ def create
 
     redirect_to disputes_strike_path(@strike), notice: I18n.t('disputes.strikes.appealed_msg')
   rescue ActiveRecord::RecordInvalid
+    flash.now[:alert] = I18n.t('disputes.strikes.appeal_error')
+    @appeal = @strike.appeal || @strike.build_appeal
```

Comment:
You can get the record from the error, that way you don’t need a flash message to render errors